### PR TITLE
doc - bump version in the static files - update doc on Gloo Admin UI deployment

### PR DESCRIPTION
**Pull request by bcollard**
Add a paragraph in the Gloo Enterprise installation guide to clarify the settings required to have the Gloo admin UI

Doc only

### CODE REVIEW

**Review by jenshu — DISMISSED** [review]

File: docs/content/installation/enterprise/_index.md [hunk 1]
```diff
@@ -226,6 +211,37 @@ Gloo Edge Open Source Helm values in Enterprise must be prefixed with `gloo`, un
 | observability.customGrafana.apiKey                        | string   | authenticate to the custom grafana instance using this api key
 | observability.customGrafana.url                           | string   | the URL for the custom grafana instance
 ---
+
+## Enterprise UI
+
+Prior to Gloo Edge Enterprise v1.8.9, the Enterprise UI was only available if Gloo Federation was enabled.
+Starting in v1.8.9, the UI is included by default for all Gloo Edge Enterprise users as well.
```
Comment (artberger) [inline docs/content/installation/enterprise/_index.md:218]: For version 1.8.9 and later, Gloo Edge Enterprise includes the user interface (UI) by default. Prior versions have the UI only when you enable Gloo Federation.

File: docs/content/installation/enterprise/_index.md [hunk 2]
```diff
@@ -226,6 +211,37 @@ Gloo Edge Open Source Helm values in Enterprise must be prefixed with `gloo`, un
...
+
+Note that if you have Gloo Federation enabled, the UI does not show any data until you [register one or more clusters]({{< versioned_link_path fromRoot="/guides/gloo_federation/cluster_registration/" >}}). If Gloo Federation is disabled, the UI shows the installed Gloo Edge instance automatically without cluster registration.
```
Comment (artberger) [inline docs/content/installation/enterprise/_index.md:220]:
```suggestion
Note that when you enable Gloo Federation in version 1.8.9 or later, the UI does not show any data until you [register one or more clusters]({{< versioned_link_path fromRoot="/guides/gloo_federation/cluster_registration/" >}}). If you do not use Gloo Federation, the UI shows the installed Gloo Edge instance automatically without cluster registration.
```

File: docs/content/installation/enterprise/_index.md [hunk 3]
```diff
@@ -226,6 +211,37 @@ Gloo Edge Open Source Helm values in Enterprise must be prefixed with `gloo`, un
...
+
+To disable the Gloo Federation install, you can set
+```shell script
+gloo-fed.enabled=false
+```
```
Comment (artberger) [inline docs/content/installation/enterprise/_index.md:225]:
```suggestion
To disable Gloo Federation, you can set `gloo-fed.enabled=false` during installation as shown in the following examples.
```

File: docs/content/static/content/version_gee_latest.md [hunk 4]
```diff
@@ -1 +1 @@
-1.9.4
\ No newline at end of file
+1.10.2
```
Comment (artberger) [inline docs/content/static/content/version_gee_latest.md:1]: A new [version](https://github.com/solo-io/gloo/releases/tag/v1.10.3) was released.
```suggestion
1.10.3
```

File: docs/content/static/content/version_geoss_latest.md [hunk 5]
```diff
@@ -1 +1 @@
-1.9.5
\ No newline at end of file
+1.10.1
```
Comment (artberger) [inline docs/content/static/content/version_geoss_latest.md:1]: [Similarly](https://github.com/solo-io/solo-projects/releases/tag/v1.10.3)
```suggestion
1.10.3
```

**Review by artberger — COMMENTED** [review]
Thanks for raising this. A few doc comments and version bump.

**Comment by bcollard** [comment]
now messy, my bad. I'll close this PR and create a new one

**Pull request closed without merging**